landlab/components/landslides/landslide_probability.py

```diff
@@ -0,0 +1,535 @@
+#!/usr/env/python
+
+"""
+landslide_probability.py:
+
+Landlab component that simulates relative wetness and probability of failure.
+
+Relative wetness and factor-of-safety are based on the infinite slope
+stability model driven by topographic and soils inputs and recharge provided
+by user in a "landslide_driver" file. In addition, the component simulates
```

Comment:
We have included a reference manual and the paper citation as references. This should take care of users' ambiguity regarding this matter.